### ISSUE TITLE
Rename the docs for switching to APM Integration

### ISSUE THREAD

**Issue by simitt** [elastic/apm-server#8252]
In [Upgrade to the Elastic APM integration](https://www.elastic.co/guide/en/apm/guide/7.17/upgrade-to-apm-integration.html) the docs use the terminology _upgrade_ for switching from APM standalone binary to managed by Elastic Agent. It is not really describing an upgrade, but rather how to move over to a new way of deployment. 
This created some confusion in the context of cloud, where an Integrations Server was introduced in `8.0`, often understood as upgrading to `8.0` would upgrade to the Elastic APM Integration, which is not automatically the case. 

@bmorelli25 could we rename the docs to _Switching to the Elastic APM integration_ instead? 

**Affected docs versions**
`7.16`,`7.17`
`8.0`,`8.1`,`8.2`

**Comment by insukcho** [comment]
Hi, team. I'm reopening this issue while I can see we might need to improve more to get rid of confusion.

- [ ] the first section subject is `Why upgrade?`. It might be better to use `Why switch` or just say something like`Why`/`Rationales` .

- [ ] the URL is still holding the word `upgrade` as `https://www.elastic.co/guide/en/apm/guide/7.16/upgrade-to-apm-integration.html`. Is it possible to change this URL, too? I guess, we just can change the HTML file name but it might break some callee links.

- [ ] [7.16.0 Release note](https://www.elastic.co/guide/en/apm/guide/7.16/release-notes-7.16.html#release-notes-7.16.0) mention this is `Upgrade guide`. Let's change to `Switch guide`.
<img width="771" alt="image" src="https://user-images.githubusercontent.com/3095667/174888359-a3d3b951-4e4b-4a69-89d0-78315a0d6ad4.png">

**Comment by bmorelli25** [comment]
Thanks for reopening this issue, Insuk. It looks like these files changed a bit in 8.0, so my backports to 7.17 and 7.16 missed these fixes. I've opened https://github.com/elastic/apm-server/pull/8431 to address your comments.

In addition, I noticed a few more instances where we say "upgrade" in versions 8.0+, so I've opened https://github.com/elastic/apm-server/pull/8430 to fix.

> the URL is still holding the word upgrade as https://www.elastic.co/guide/en/apm/guide/7.16/upgrade-to-apm-integration.html. Is it possible to change this URL, too?

This is a bit more involved to change as this page is linked to from a lot of different places in the docs. How important is changing this URL?
